eth-price-oracle-client/README.md

```diff
@@ -45,34 +45,37 @@ cd eth-price-oracle-client
 npm install
 ```
 
-4. Create a `.env` file in the root directory with your private key:
+4. Create a `.env` file in the root directory with your private keys:
 ```
-PRIVATE_KEY=your_private_key_here
+PRIVATE_KEY_1=your_private_key_1_here
+PRIVATE_KEY_2=your_private_key_2_here
+PRIVATE_KEY_3=your_private_key_3_here
 ```
 
 ## Usage
 
 Run the client with default settings:
 ```bash
-npm run dev
+npm run dev -- --strategy 19 --key_number 1
 ```
 
 Run with specific strategy and calculation type:
 ```bash
-npm run dev -- --strategy 19 --calculation_type arithmetic
+npm run dev -- --strategy 19 --calculation_type arithmetic --key_number 1
```

Comment:
how about:

`PRIVATE_KEY=$PRIVATE_KEY_3 npm run dev -- --strategy 19 --calculation_type arithmetic`

This way you are overriding the value of `PRIVATE_KEY` for the current session, and the code **always** looks up `PRIVATE_KEY` env?
This way it's scalable, for 1 to n private keys, it is not linked to any specific variable name other than the one, and the code does not have `if key_number == 1 then...`